README.md

```diff
@@ -0,0 +1,13 @@
+# Survey Form
+
```

Comment:
Kindly consider not removing any relevant section from the Readme template. for example, the Future Features section is missing from your readme, so consider adding that to your readme.

Reply:
You did really well adding more inline comments and giving a polite review as suggested by the previous reviewer, please as a part of a good communication structure in reviews, you are to give a reason `why` you are suggesting such changes from the student to enable them to know the importance of them implementing the changes you requested for. You did that in your first review, please include it in your new reviews 😄